Description: Vala 0.41.92 does not allow custom getter default values
Author: fossfreedom <fossfreedom@ubuntu.com>
Commit: https://github.com/UbuntuBudgie/budgie-desktop/commit/c7be4c220762116c251258d4c840eb75de026f2c
Last Update: 2018-08-26

---
 src/applets/icon-tasklist/Icon.vala    | 10 +++++-----
 src/applets/status/PowerIndicator.vala |  2 +-
 src/panel/panel.vala                   |  2 +-
 src/raven/headerwidget.vala            |  2 +-
 src/raven/raven.vala                   |  2 +-
 5 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/src/applets/icon-tasklist/Icon.vala b/src/applets/icon-tasklist/Icon.vala
index 895cd38c..e0610d5a 100644
--- a/src/applets/icon-tasklist/Icon.vala
+++ b/src/applets/icon-tasklist/Icon.vala
@@ -20,8 +20,8 @@ public class Icon : Gtk.Image
     private int wait_cycle_counter = 0;
     private int attention_cycle_counter = 0;

-    private double bounce_amount = 0;
-    private double attention_amount = 0;
+    private double bounce_amount = 0.0;
+    private double attention_amount = 0.0;

     public double bounce {
         public set {
@@ -31,7 +31,7 @@ public class Icon : Gtk.Image
         public get {
             return bounce_amount;
         }
-        default = 0.0;
+        //default = 0.0;
     }

     public double attention {
@@ -42,7 +42,7 @@ public class Icon : Gtk.Image
         public get {
             return attention_amount;
         }
-        default = 0.0;
+        //default = 0.0;
     }

     public double icon_opacity {
@@ -56,7 +56,7 @@ public class Icon : Gtk.Image
         public get {
             return opacity;
         }
-        default = 1.0;
+        //default = 1.0;
     }

     public Icon() {}
diff --git a/src/applets/status/PowerIndicator.vala b/src/applets/status/PowerIndicator.vala
index 1816a294..6c498968 100644
--- a/src/applets/status/PowerIndicator.vala
+++ b/src/applets/status/PowerIndicator.vala
@@ -30,7 +30,7 @@ public class BatteryIcon : Gtk.Box
         public get {
             return this.percent_label.visible;
         }
-        default = false;
+        //default = false;
     }

     public BatteryIcon(Up.Device battery) {
diff --git a/src/panel/panel.vala b/src/panel/panel.vala
index 17bd9431..2b73a287 100644
--- a/src/panel/panel.vala
+++ b/src/panel/panel.vala
@@ -120,7 +120,7 @@ public class Panel : Budgie.Toplevel
         public get {
             return render_scale;
         }
-        default = 0.0;
+        //default = 0.0;
     }

     public bool activate_action(int remote_action)
diff --git a/src/raven/headerwidget.vala b/src/raven/headerwidget.vala
index 429c350c..97bbe2c7 100644
--- a/src/raven/headerwidget.vala
+++ b/src/raven/headerwidget.vala
@@ -34,7 +34,7 @@ public class HeaderExpander : Gtk.Button
         public get {
             return this._expanded;
         }
-        default = false;
+        //default = false;
     }

     public HeaderExpander(HeaderWidget? owner)
diff --git a/src/raven/raven.vala b/src/raven/raven.vala
index 23250d52..47ad19e1 100644
--- a/src/raven/raven.vala
+++ b/src/raven/raven.vala
@@ -170,7 +170,7 @@ public class Raven : Gtk.Window
         public get {
             return this._screen_edge;
         }
-        default = Gtk.PositionType.RIGHT;
+        //default = Gtk.PositionType.RIGHT;
     }

     int our_width = 0;
--
2.17.1
